core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -143,7 +143,7 @@
  * @see AbstractBuild
  */
 @SuppressWarnings("rawtypes")
-public abstract class AbstractProject<P extends AbstractProject<P,R>,R extends AbstractBuild<P,R>> extends Job<P,R> implements BuildableItem, LazyBuildMixIn.LazyLoadingJob<P,R>, ParameterizedJobMixIn.ParameterizedJob {
+public abstract class AbstractProject<P extends AbstractProject<P,R>,R extends AbstractBuild<P,R>> extends Job<P,R> implements BuildableItem, LazyBuildMixIn.LazyLoadingJob<P,R>, ParameterizedJobMixIn.ParameterizedJob<P, R> {
```

Comment:
`implements BuildableItem` is now redundant, could be removed I guess.

> I have checked all usages of "instanceof BuildableItem".

The only implementations of `ParameterizedJob` were already `BuildableItem`s so retrofitting it to implement that interface should not cause any behavioral change even if someone _were_ doing that `instanceof` check.